src/NuGet.Core/NuGet.ProjectModel/PackageSpec.cs

```diff
@@ -162,6 +162,18 @@ public NuGetVersion Version
         /// <remarks>Optional. This is normally set for internal use only.</remarks>
         public ProjectRestoreMetadata RestoreMetadata { get; set; }
 
+        /// <summary>
+        /// A unified flag to help users manage their SDK warning levels. Example: 9.0.100
+        /// When introducing a new warning or error use this property to
+        /// allow users to tell the sdk "treat me as if I were SDK x.y.z" and manage breaking changes
+        /// </summary>
+        public NuGetVersion SdkAnalysisLevel { get; set; }
```

Comment:
I don't think this is a top level thing. 

https://github.com/NuGet/NuGet.Client/blob/20f05435be385abfe74737b6433dc80fd3b3b504/src/NuGet.Core/NuGet.ProjectModel/ProjectRestoreMetadata.cs should contain this instead. 

You'll see other unique values being represented there.